iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -43,12 +43,7 @@ sealed class MqttTransportHandler : TransportHandler, IMqttIotHubEventHandler
 
         readonly string generationId = Guid.NewGuid().ToString();
 
-        static readonly ConcurrentObjectPool<string, IEventLoopGroup> EventLoopGroupPool =
-            new ConcurrentObjectPool<string, IEventLoopGroup>(
-                Environment.ProcessorCount,
-                () => new MultithreadEventLoopGroup(eg => new SingleThreadEventLoop(eg, "MQTTExecutionThread", TimeSpan.FromSeconds(1)), 1),
-                TimeSpan.FromSeconds(5),
-                elg => elg.ShutdownGracefullyAsync());
+        private static MultithreadEventLoopGroup s_eventLoopGroup = new MultithreadEventLoopGroup();
```

Comment:
Be aware that this will start `Environment.ProcessorCount * 2` number of threads immediately. Not sure if there is a concern with this on smaller devices.

Reply:
I'll leave it as-is for now: In a future SDK version we should expose the handler and the detailed configuration.

Reply:
Added as comments.